tests/UI/campaigns/functional/FO/04_contactUs/01_checkMailtoLInk.js

```diff
@@ -0,0 +1,62 @@
+require('module-alias/register');
+
+const {expect} = require('chai');
+
+const helper = require('@utils/helpers');
+
+// Import pages
+const homePage = require('@pages/FO/home');
+const contactUsPage = require('@pages/FO/contactUs');
+
+// Import data
+const {DefaultEmployee} = require('@data/demo/employees');
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_FO_headerAndFooter_checkLinksInHeader';
```

Comment:
```suggestion
const baseContext = 'functional_FO_contactUs_checkMailtoLink';
```

:warning: @boubkerbribri the name of the file has LInk and not Link ;)

Reply:
done thanks